tests/unit/main.ts

```diff
@@ -58,4 +67,18 @@ describe('main', () => {
 			assert.strictEqual(writeFileStub.getCall(0).args[0], 'test.css.d.ts');
 		});
 	});
+
+	it('should create a Stylus CSS module declaration file', () => {
+		const writeFileStub: sinon.SinonStub = sandbox.stub(fs, 'writeFile', function (name: string, content: string, done: Function) {
```

Comment:
might look a bit tidier if you imported `{ SinonStub }` from sinon as well as the default.